main.nf

```diff
@@ -488,212 +755,622 @@ if (params.adna){
             samtools view -h -F 4 $bam2 | pmdtools -t ${params.pmdscore} --header $library | samtools view -Sb - > $outfile
             """
     }
+
+    if (params.name3 != ''){
+        process pmdtoolsgenome3 {
+        tag "$name"
+
+        label 'ristretto'
+
+        publishDir "${params.outdir}/pmdtools/${params.name3}", mode: 'copy', pattern: '*.pmd_filtered.bam'
+
+        input:
+            set val(name), file(bam3) from alignment_genome3
+        output:
+            set val(name), file("*.pmd_filtered.bam") into pmd_aligned3
+        script:
+            outfile = name+"_"+params.name3+".pmd_filtered.bam"
+            """
+            samtools view -h -F 4 $bam3 | pmdtools -t ${params.pmdscore} --header $library | samtools view -Sb - > $outfile
+            """
+        }
+    }   
 }
 
-// 4:   Count aligned bp on each genome and compute ratio
+process kraken2 {
+    tag "$name"
+
+    label 'intenso'
+
+    input:
+        set val(name), file(reads) from unmapped_humans_reads
+        file(krakendb) from krakendb_ch
+
+
+    output:
+        set val(name), file('*.kraken.out') into kraken_out
+        set val(name), file('*.kreport') into kraken_report
+
+    script:
+        out = name+".kraken.out"
+        kreport = name+".kreport"
+        if (pairedEnd && params.collapse == false){
+            """
+            kraken2 --db ${krakendb} \
```

Comment:
Do single `\` work? Ive always used `\\`. Cant find any documentation anywhere!

Reply:
Both work, but I agree, `\\` is better for looking at the `.command.sh` after.
Single `\` adds too much spacing between flags.
Changed with #7